qemu/tests/format_disk.py

```diff
@@ -33,7 +33,8 @@ def run(test, params, env):
     os_type = params["os_type"]
 
     if os_type == 'linux':
-        session.cmd("dmesg -C")
+        dmesg_cmd = params.get("dmesg_cmd")
```

Comment:
What about dmesg_cmd = params.get("dmesg_cmd", "dmesg -C")？

Then in your format_disk.cfg, you just need to set:
   RHEL.6:
        fs_type = ext4
        dmesg_cmd = "dmesg -c"

Reply:
Thanks @aliang123 , updated ! please help to review again :)

Reply:
Hi @aliang123 corrected, thanks !